prow/plugins/override/override.go

```diff
@@ -112,7 +112,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		Description: "Forces a github status context to green (one per line).",
 		Featured:    false,
 		WhoCanUse:   "Repo administrators",
-		Examples:    []string{"/override pull-repo-whatever", "/override continuous-integration/travis\n/override deleted-job"},
+		Examples:    []string{"/override pull-repo-whatever", "/override ci/circleci", "/override deleted-job"},
```

Comment:
- the \n did not work here at all
- the other change makes this much shorter, and it gets across the same concept